serverless_template.yml

```diff
@@ -107,12 +109,12 @@ functions:
     events:
       - http:
           method: get
-          path: images/generate
+          path: generate
```

Comment:
Now when we generate an image we'll redirect users to `images.image-service-<stage>.pinster.io/generate` and then that will eventually redirect them to `image-service-<stage>.pinster.io/<imageName>`

Reply:
👍